websites/api-docs-smoke-test/package.json

```diff
@@ -10,8 +10,8 @@
     "clean": "npx gatsby clean",
     "clean:develop": "yarn clean && yarn develop",
     "build": "npx gatsby build --prefix-paths",
-    "prebuild": "yarn clean && yarn predevelop",
-    "postbuild": "rm -rf ../../public/api-docs-smoke-test && mkdir -p ../../public && mv public ../../public/api-docs-smoke-test",
+    "prebuild": "mkdir -p ../../public/api-docs-smoke-test && rm -rf public && mv ../../public/api-docs-smoke-test ./public && mkdir -p ../../.cache/api-docs-smoke-test && rm -rf .cache && mv ../../.cache/api-docs-smoke-test ./.cache && yarn predevelop",
+    "postbuild": "mv public ../../public/api-docs-smoke-test && mv .cache ../../.cache/api-docs-smoke-test",
```

Comment:
This is the other required step:  once Vercel has detected gatsyby, it assumes the top level `.cache` and `public` folders to be the cacheable places. 

So we move the site-specific caches into the global location (into a subfolder with the site name) and back.  For `public` this was done anyways after the build.  

It would be nicer if this job would be done centrally and more generic though - maintaining all those long script commands across many sites like in the content repos is error prone (but not super hard either because they don't change).  Maybe a little node script?  

If it were central, we could also set it up so that it only runs on vercel, not locally.